RxSwiftWorkshop/Maps/Model/GeolocatorSpec.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import RxSwift
+import CoreLocation
+
+@testable
+import RxSwiftWorkshop
+
+class GeolocatorSpec: QuickSpec {
+
+    override func spec() {
+        describe("Geolocator") {
+
+            var disposeBag: DisposeBag?
+            var systemGeolocator: FakeSystemGeolocator?
+            var sut: Geolocator?
+
+            beforeEach {
+                disposeBag = DisposeBag()
+                let geolocator = FakeSystemGeolocator()
+                systemGeolocator = geolocator
+                sut = Geolocator(geolocator: geolocator)
+            }
+
+            afterEach {
+                disposeBag = nil
+                systemGeolocator = nil
+                sut = nil
+            }
+
+            it("should pass geolocation request to its SystemGeolocator") {
+                guard case let (disposeBag?, systemGeolocator?) = unwrap(disposeBag, systemGeolocator) else { return }
```

Comment:
`unwrap` is pretty neat 👍 
But it doesn't unwrap actually... right? 

I don't have better name for now.